src/shared/configs/axios/index.ts

```diff
@@ -1,22 +1,41 @@
 import Axios, { AxiosError, AxiosResponse } from 'axios';
+import { GetServerSidePropsContext } from 'next';
 
-import { getAccessToken } from '../cookie';
+import { accessTokenName, getAccessToken } from '../cookie';
 
+import isServer from '@src/shared/utils/isServer';
 import { BaseRes, MemoChatError } from '@src/shared/types/api';
 
 const axios = Axios.create({
   baseURL: 'https://memochat-server.herokuapp.com/v1',
 });
 
+let serverSideCookies: GetServerSidePropsContext['req']['cookies'];
+
+export const setServerSideToken = (cookies: GetServerSidePropsContext['req']['cookies']) => {
+  serverSideCookies = cookies;
+};
+
 axios.interceptors.request.use(
   (config) => {
-    const accessToken = getAccessToken();
-    if (accessToken) {
-      config.headers = {
-        Authorization: `Bearer ${accessToken}`,
-      };
+    if (isServer()) {
+      const serverAccessToken = serverSideCookies[accessTokenName];
```

Comment:
isServer를 따로 분기치치 않고 이렇게 수정해도 되지않을까욥?
```suggestion
      const accessToken = isServer()? serverSideCookies[accessTokenName]: getAccessToken();
```

Reply:
e20e5b9a37a12689a0180370343dd5c299e7d98a